src/data/actions/assets.test.jsx

```diff
@@ -159,15 +206,86 @@ describe('Assets Action Creators', () => {
     expect(store.dispatch(actionCreators.uploadExceedMaxCount(1))).toEqual(expectedAction);
   });
   it('returns expected state from uploadAssetFailure', () => {
-    const expectedAction = { file: 'file', response: 'response', type: assetActions.UPLOAD_ASSET_FAILURE };
+    const expectedAction = { asset: 'file', response: 'response', type: assetActions.UPLOAD_ASSET_FAILURE };
     expect(store.dispatch(actionCreators.uploadAssetFailure('file', 'response'))).toEqual(expectedAction);
   });
   it('returns expected state from uploadAssetSuccess', () => {
-    const expectedAction = { asset: 'asset', response: 'response', type: assetActions.UPLOAD_ASSET_SUCCESS };
-    expect(store.dispatch(actionCreators.uploadAssetSuccess('asset', 'response'))).toEqual(expectedAction);
+    const expectedAction = { response: 'response', type: assetActions.UPLOAD_ASSET_SUCCESS };
+    expect(store.dispatch(actionCreators.uploadAssetSuccess('response'))).toEqual(expectedAction);
```

Comment:
Just changing the test to match the changes I made to the uploadAssetSuccess action creator.